db/dataset_eval.py

```diff
@@ -296,6 +296,28 @@ def _create_job(connection, dataset_id, normalize, eval_location, filter_type=No
     return job_id
 
 
+def get_user_pending_jobs(user_api_key):
+    with db.engine.connect() as connection:
+        query = sqlalchemy.text("""
+                    SELECT dataset_eval_jobs.id
```

Comment:
double check the way that we format SQL statements. This should be consistent throughout the file